src/shell/lisa_gen_buildroot_rootfs

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -eu
+
+BUILDROOT_URI=git://git.busybox.net/buildroot
+BUILDROOT_VERSION=2018.08
+BUILDROOT_DIR=$LISA_HOME/src/buildroot
+BUILDROOT_PATCHES_DIR=$(dirname $0)/buildroot_patches
```

Comment:
$0 needs quotes: "$0", otherwise word splitting will occur and you could end up passing multiple parameters to dirname, each getting their output line in dirname output.

Reply:
nit: $BASH_SOURCE is a handy replacement for $0, and has the advantage of always pointing to the current file even when the script is sourced (but it is obviously not POSIX).

Reply:
Yeah we are not guaranteed that the user won't have put lisa in a directory with spaces in the name - I'll fix all similar occurrences up

Reply:
drat; buildroot (or maybe make) doesn' thandle the space in the path anyway - oh well there's so much one can do \o/